x-pack/test_serverless/api_integration/test_suites/observability/ai_assistant/tests/knowledge_base/knowledge_base_status.spec.ts

```diff
@@ -70,10 +58,8 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       await deleteInferenceEndpoint({ es, name: AI_ASSISTANT_KB_INFERENCE_ID });
 
       const res = await observabilityAIAssistantAPIClient
-        .slsUser({
+        .slsAdmin({
```

Comment:
If admin privileges are required to access the status endpoint, what happens when users with the `editor` role call it? The status endpoint is called when starting a conversation to ensure the KB has been setup. Do we have a problem there?